docs/the-new-architecture/pillars.md

```diff
@@ -21,7 +21,8 @@ For the sake of this guide we're going to use the following **terminology**:
 
 - **Legacy Native Components** & **Legacy Native Modules** - To refer to Modules and Components which are running on the old React Native architecture.
 - **Fabric Native Components** & **Turbo Native Modules** - To refer to Modules and Components which have been adapted to work well with the New Architecture, namely the new renderer and the new Native Module System. For brevity you might find them referred as **Fabric Components** and **Turbo Modules**
-  :::
+
```

Comment:
Empty line is needed to prevent `yarn lint --fix` from messing up the formatting in the future.